.github/workflows/build-and-test.yml

```diff
@@ -83,8 +88,38 @@ jobs:
                   python -m pip install -r requirements.txt -r requirements-dev.txt
                   python setup.py install
 
+            - name: Set up MacOS python
+              if: matrix.os != 'windows-latest'
+              uses: goanpeca/setup-miniconda@v1.1.2
+              with:
+                  update-conda: false
+                  activate-environment: env
+                  auto-update-conda: true 
+                  python-version: ${{ matrix.python-version }}
+                  conda-channels: defaults
+                  channels: conda-forge, anaconda
```

Comment:
Hey @davemfish , this looks likes some duplication between `setup-conda` (which we used to use) and `setup-miniconda` (which we've decided to use going forward). I believe `setup-conda` uses `conda-channels` and `setup-miniconda` uses `channels`. So you could probably drop `conda-channels` here.  Similarly I think that `update-conda` is from `setup-conda` and that `auto-update-conda` is `setup-miniconda`s equivalent. So `update-conda` could be removed as it's probably not being used...

Sorry if I'm missing something and feel free to ignore...

Reply:
Thanks, @dcdenu4 ! You're input always appreciated. Indeed, this section needs cleaning up.